pkg/cli/cli.go

```diff
@@ -25,6 +25,12 @@ func Run(configPath string, inputPath string, outputPath string) error {
 			obfuscators = append(obfuscators, obfuscator.NewKeywordsObfuscator(o.Replacement))
 		case schema.ObfuscateTypeMAC:
 			obfuscators = append(obfuscators, obfuscator.NewMacAddressObfuscator())
+		case schema.ObfuscateTypeRegex:
+			o, err := obfuscator.NewRegexObfuscator(*o.Regex, string(o.Target))
```

Comment:
the schema already generated
> type ObfuscateTarget string

let's use the typedef instead, so people are clear those are enums somehow

Reply:
Done